### PR TITLE
Convert TDC to tick for true energy deposition.

### PR DESCRIPTION
**Pull request by yangtj207**
While testing this code on SBND,  Mun Jung and I noticed a need to convert the tdc value in SimChannel to the electronics time tick in order to be comparable with the real readout. Now this conversion is properly done so this code should work for all detectors. Thank you to Dom Brailsford for pointing this problem out.

### CODE REVIEW

**Comment by yangtj207** [comment]
trigger build

File: larrecodnn/ImagePatternAlgs/Tensorflow/PointIdAlg/PointIdAlg.cxx [hunk 1]
```diff
@@ -1013,13 +1013,12 @@ nnet::TrainingDataAlg::setEventData(const art::Event& event,
...
+      
+      int tick_idx = clockData.TPCTDC2Tick(ttc.first) + fAdcDelay;
 
-      if (ttc.first < labels_size) {
-        int tick_idx = ttc.first + fAdcDelay;
-        if (tick_idx < labels_size) {
-          labels_deposit[tick_idx] = max_deposit;
-          labels_pdg[tick_idx] = max_pdg & type_pdg_mask;
-        }
+      if (tick_idx < labels_size && tick_idx > 0) {
```
Comment (knoepfel) [inline larrecodnn/ImagePatternAlgs/Tensorflow/PointIdAlg/PointIdAlg.cxx:1019]: Is there ever a case where `tick_idx` should be `0`?

Comment (yangtj207) [reply larrecodnn/ImagePatternAlgs/Tensorflow/PointIdAlg/PointIdAlg.cxx]: Yes, 0 should be allowed. I've updated it. Thanks.

**Review by knoepfel — COMMENTED** [review]
Changes look good, Tingjun.  Thanks.

**Comment by lgarren** [comment]
trigger build

**Comment by lgarren** [comment]
@LArSoft/level-2-managers All tests look good.  The DUNE test failure is unrelated.  Do you approve this PR?

**Comment by yangtj207** [comment]
approve

**Comment by lgarren** [comment]
approve

**Pull request merged**